packages/ingest/s3.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const aws = require('@cumulus/common/aws');
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const urljoin = require('url-join');
+
+module.exports.s3Mixin = superclass => class extends superclass {
+
+  /**
+   * Downloads a given url and upload to a given S3 location
+   * @return {Promise}
+   * @private
+   */
+  async sync(_path, bucket, key, filename) {
```

Comment:
The first argument is used, so it should not start with an underscore.  An underscore indicates an argument that must be present for positional reasons, but is not used.